server/src/endpoints/store.rs

```diff
@@ -36,7 +36,7 @@ enum BadStoreRequest {
     #[fail(display = "failed to fetch project information")]
     ProjectFailed(#[cause] ProjectError),
 
-    #[fail(display = "failed to process event")]
+    #[fail(display = "failed to process event: {}", _0)]
```

Comment:
With `cause`, this should not be necessary. The inner error will be rendered as “\n caused by: ...”